source/extensions/filters/listener/proxy_protocol/proxy_protocol.h

```diff
@@ -43,6 +43,17 @@ using ConfigSharedPtr = std::shared_ptr<Config>;
 
 enum ProxyProtocolVersion { Unknown = -1, InProgress = -2, V1 = 1, V2 = 2 };
 
+enum class ProxyState{
+    //failed to read proxy protocol
+    PrtclFail,
```

Comment:
Can you give this a name without abbreviations for clarity? `ProtocolFail`? `Error`?

Reply:
I've changed them to "ProtocolFail" in both .h and .cc files